docs/reference/changelog.rst

```diff
@@ -4,6 +4,24 @@
 Changelog
 *********
 
+X.Y.Z (2024-MMM-DD)
```

Comment:
```suggestion
X.Y.0 (2024-MMM-DD)
```

Not sure how you feel about this, but that indicates that this branch now contains new features (so we can no longer make hotfixes off main).